Chinese/迪力.txt

```diff
@@ -0,0 +1 @@
+迪力会成为一名伟大的程序员，他正在为这个梦想不断奋斗着。继续勇往直前吧，Dilyar!
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.